### PR TITLE
Updated launch file

### PR DESCRIPTION
**Pull request by DLu**
Spurred by [this conversation](https://github.com/ros2/launch_ros/pull/268) I've updated the launch file here to use `ParameterValue` and while I was at it, updated a few other things.

### CODE REVIEW

File: launch/display.launch.py [hunk 1]
```diff
@@ -1,48 +1,58 @@
...
-                                             description='Flag to enable joint_state_publisher_gui'),
-        launch.actions.DeclareLaunchArgument(name='model', default_value=default_model_path,
-                                             description='Absolute path to robot urdf file'),
-        launch.actions.DeclareLaunchArgument(name='rvizconfig', default_value=default_rviz_config_path,
-                                             description='Absolute path to rviz config file'),
-        joint_state_publisher_node,
-        joint_state_publisher_gui_node,
-        robot_state_publisher_node,
-        rviz_node
-    ])
```
Comment (hidmic) [inline launch/display.launch.py]: @DLu meta: I have to say I do have a slight preference for the _"single statement"_ style, but this is an improvement nonetheless.

**Comment by DLu** [comment]
Sorry, wrong thread.

**Review by hidmic — APPROVED** [review]
Tested this locally, LGTM!

File: launch/display.launch.py (hunk 1 above)
Comment (DLu) [reply launch/display.launch.py]: I like having my argument declarations at the top, so let's compromise.

**Review by hidmic — APPROVED** [review]

**Pull request merged**